### ISSUE TITLE
Handling votekicks

### ISSUE THREAD

**Issue by NotAFile** [piqueserver/aosprotocol#6]
It would be great to have some way to show votes client side. This will probably increase participation as well as generally feel better.

It might be an option to make this a text packet.

**Comment by feikname** [comment]
This certainly would increase participation, and is a feature I've been longing for ever since AoS started. Totally in favor.

We can use [OpenSpades's Client Features Flags](https://github.com/yvt/openspades/wiki/Network-Protocols#version-info-payload-2-client-feature-flags-1-since-011) to detect whether the client supports GUI votes or not (might be a good idea to implement other kinds of votes together as well). If not, fall back to text mode.

**Comment by NotAFile** [comment]
Yes, the Feature flags are requirements for pretty much all new packets we add...

However, we really must do feature flags on both sides, for other use cases. Adding e.g. #2 would require the Server telling the client it supports additional packets.

That means for me, that the "feature flag" thing really won't scale... we need to add one flag per packet and can add 31 new features before exhausting the flags. Negotiating some kind of "protocol sub-version" might be a better idea and then doing the mapping from that to supported features client and server side.

**Comment by noway** [comment]
On a more general to this faeture note, I think custom gui boxes is what needed. I.e. the ability to manage UI from server and create special blocks of text such as "2/21\nVoted"
